src/Hosting/README.md

```diff
@@ -0,0 +1,31 @@
+# Hosting
+
+Hosting provides the entry point for configuring and running your application.
+
+Documentation for Hosting can be found in [ASP.NET Core Web Host](https://docs.microsoft.com/aspnet/core/fundamentals/host/web-host) and [.NET Generic Host in ASP.NET Core](https://docs.microsoft.com/aspnet/core/fundamentals/host/generic-host) docs.
+
+## Description
+
+The following contains a description of the sub-directories.
+
+`Abstractions`: Contains the main Hosting and Startup interfaces.
+`Hosting`: Contains the default implementations for GenericHost, WebHost, and Startup.
+`Server.Abstractions`: Contains the interfaces for Server implementations.
+`Server.IntegrationTesting`: Contains classes to help deploy servers for testing across the repo.
+`TestHost`: Contains a server implementation for in-memory testing against a server.
+`WindowsServices`: Contains methods to run an application as a Windows service.
+`samples`: Contains a few sample apps that show examples of using hosting.
+
+## Development Setup
+
+### Build
+
+To build this specific project from source, you can follow the instructions [on building a subset of the code](../../BuildFromSource.md#building-a-subset-of-the-code).
```

Comment:
```suggestion
To build this specific project from source, you can follow the instructions [on building a subset of the code](../../docs/BuildFromSource.md#building-a-subset-of-the-code).
```

Reply:
This deep link skips all of the pre-reqs. That's going to be confusing.

Reply:
Both these comments should be raised on the README issue since these sections are copied from the template there.

Reply:
Yep. Check out https://github.com/dotnet/aspnetcore/pull/27976 where I gave the master doc a spruce up.